src/app/routes/features/features.component.ts

```diff
@@ -17,6 +18,51 @@ export class FeaturesComponent implements OnInit {
   }
 
   ngOnInit() {
+
+      const properties = new Map<string, Property>();
```

Comment:
```suggestion
      const properties = new Map<string, Property>();
      properties.set('Sample Property', {
          'name': 'SampleProperty',
          'description': '',
          'type': 'org.ff4j.property.PropertyString',
          'value': 'go!',
          'fixedValues': []
      });
       const initParams = new Map<string, string>();
      initParams.set('expression', 'f1 & f2 | !f1 | f2');
       this.features = [{
          uid: 'Feature_UID',
          description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
          magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
          consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
          est laborum.`,
          group: 'Group Name',
          enable: false,
          permissions: [ 'ROLE_ADMIN', 'ROLE_USER', 'ROLE_ADMIN', 'ROLE_USER' ],
          customProperties: properties,
          flippingStrategy: {
              type: 'org.ff4j.strategy.el.ExpressionFlipStrategy',
              initParams: initParams
          }
      },
          {
              uid: 'Feature_UID',
              description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
          magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
          consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
          est laborum.`,
              group: 'Group Name',
              enable: false,
              permissions: [ 'ROLE_ADMIN', 'ROLE_USER', 'ROLE_ADMIN', 'ROLE_USER' ],
              customProperties: properties,
              flippingStrategy: {
                  type: 'org.ff4j.strategy.el.ExpressionFlipStrategy',
                  initParams: initParams
              }
          }];
```

is not needed you would have to start with the mock server

Reply:
In fact, I tried but I still got the error. I ll check once more and remove this.